app/controllers/admin/events_controller.rb

```diff
@@ -92,7 +95,27 @@ def update
       end
     end
 
-    def create; end
+    def create
+      @url = admin_conference_program_events_path(@conference.short_title, @event)
+      @users = User.all.order(:name)
+      @languages = @program.languages_list
+      @event.submitter = current_user
+      @event.state = :confirmed
```

Comment:
@differentreality it is up to the team whether to keep this event.state override, the code exported in this pull request is based on out internal needs at CommandPrompt, so I just decided to keep it for now, I can remove that if you wish.
Besides that, is there anything else you wish to have changed?